pyproject.toml

```diff
@@ -11,7 +11,8 @@ description = "Tooling to manage the `pyodide-lock.json` file"
 readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
-   "pydantic"
+    # compatible with pyodide-build, and the as-shipped wheel in the pyodide distributiont
```

Comment:
typos after the the ","

Reply:
Better?

Reply:
I don't really understand what you mean by "as-shipped wheel" but it's probably just me :) Fine to merge as is.

Reply:
This feller: https://github.com/pyodide/pyodide/blob/main/packages/pydantic/meta.yaml